api/HTMLMediaElement.json

```diff
@@ -2209,7 +2212,7 @@
               "version_added": "43"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "43"
```

Comment:
This copying incorrect data, a case of https://github.com/mdn/browser-compat-data/issues/7843. Can back out these changes you fix all of the "43" values in this file, updating both Chrome and mirrored data at the same time?

Reply:
This is really odd, running the collector on Chrome 1 shows support all the way back then, but the collector isn't updating this data...  I'll make a note of this in the related collector's issue.